app/services/users.js

```diff
@@ -11,3 +11,16 @@ exports.findByEmail = value => User.findOne({ where: { email: value } });
 
 exports.createUser = (firstName, lastName, email, password) =>
   User.create({ firstName, lastName, email, password });
+
+exports.listUsers = (limit, offset, page) =>
+  User.findAndCountAll({ limit, offset }).then(users => ({
+    page: users.rows,
```

Comment:
Create a serializer file to change the data for the response. Is the opposite to mapper